app/models/submission.rb

```diff
@@ -15,7 +15,28 @@ class Submission < ApplicationRecord
   has_many :submission_packets, dependent: :destroy
   has_many :poems, through: :submission_packets
 
+  accepts_nested_attributes_for :submission_packets,
+    reject_if: :empty_packet?
+
   validates :title, presence: true
-  validates :status, presence: true, inclusion: { in: SUBMISSION_STATUSES }
+  validates :status, presence: true, inclusion: { in: STATUSES }
   validates :submitted_to, presence: true
+
+  def empty_packet?(attributes)
+    attributes[:poem_attributes].each_value.all?(&:blank?)
+  end
+
+  def deep_blank?(attributes)
```

Comment:
As @jcmorrow pointed out, I would love to see some tests of this method. As is, I believe the method isn't going to do what you want it to do -- it will return `attributes`.
```rb
> {a: 1, b: { c: 3, d: 4}}.each_pair { |k, v| true }

{:a=>1, :b=>{:c=>3, :d=>4}}
```